config/initializers/highlight_override.rb

```diff
@@ -53,3 +53,9 @@ def self.create_from_model(model)
     new(model.attributes)
   end
 end
+
+Api::V0::Bindings::NewHighlight.class_exec do
```

Comment:
Wonder if this create_model! should be #create_highlight (it seems confusing that a NewHighlight creates a Highlight..just the naming seemed like it was perhaps a typo).  Also, is it necessary for this create_model! to have a bang? (the purpose of bang is to be used as a differentiator to a non-bang method, the more dangerous version of the non-bang version.  I believe it's not recommended to have a stand-along bang method.

Reply:
I could see devs reading `NewHighlight#create_highlight` as creating a Highlight binding instance (which is actually how I first read your statement "it seems confusing that a NewHighlight creates a Highlight").  `#create_model` implies to me making an AR model.  Should we go with `#create_highlight_model` to be clearer?

Like said above, the bang here for me is a way to tell the devs that this method calls `Highlight.create!` internally and to expect exceptions.  I know there's the argument about `!` only being used when there's a non-bang version that is less dangerous, but in this context I think it echoes the `save`/`save!` and `create`/`create!` activerecord methods which have never really seemed about danger but more about exceptions and performance.

Reply:
I was about to change it to `#create_highlight_model` but then I noticed that we already have `#create_from_model` so now I'm wondering if we should leave it so we have consistency in `#create_model` and `#create_from_model`.

Reply:
@jpslav agreed.  in the end..it's about consistency, readability.   these two are symmetrical and are at the same abstraction level, so sounds good.